test/generator-base-private.spec.js

```diff
@@ -269,4 +269,56 @@ export * from './entityFolderName/entityFileName.state';`;
             });
         });
     });
+    describe('getEntityParentPathAddition', () => {
+        describe('when passing /', () => {
+            it('returns empty', () => {
```

Comment:
just a quick thing: can you replace every "returns empty" by "returns an empty string"?
it's better to have explicit tests